springfox-spring-web/src/main/java/springfox/documentation/spring/web/plugins/JavaDocBuilderPlugin.java

```diff
@@ -0,0 +1,158 @@
+/*
+ *
+ *  Copyright 2015 the original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *
+ */
+package springfox.documentation.spring.web.plugins;
+
+/**
+ * Plugin to generate the @ApiParam and @ApiOperation values.
+ */
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Method;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StringUtils;
+
+import springfox.documentation.service.ResolvedMethodParameter;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spi.service.OperationBuilderPlugin;
+import springfox.documentation.spi.service.ParameterBuilderPlugin;
+import springfox.documentation.spi.service.contexts.OperationContext;
+import springfox.documentation.spi.service.contexts.ParameterContext;
+
+/**
+ * Swagger JavaDoc properties builder plugin.
+ */
+@Component
+@Order(Ordered.LOWEST_PRECEDENCE)
+public class JavaDocBuilderPlugin implements OperationBuilderPlugin, ParameterBuilderPlugin {
```

Comment:
That is correct. It would be nice to have the return information but I am not sure where to inject it. I suspect throws makes no sense since a REST service shouldn't throw anything.